image/tarexport/tarexport.go

```diff
@@ -30,18 +32,36 @@ type tarexporter struct {
 	loggerImgEvent LogImageEvent
 }
 
+type ocitarexporter struct {
+	te   *tarexporter
+	name string
```

Comment:
My best guess is because `ImportImage` means a different thing in `daemon` than what is happening in here.